src/ensemble/CavityEnsemble.cpp

```diff
@@ -290,18 +288,18 @@ unsigned CavityEnsemble::countNeighbours(ParticleContainer *container, Molecule
         lo[d] = std::max(0.0, m1->r(d) - R);
         hi[d] = std::min(system[d], m1->r(d) + R);
     }
-    //global_log->info() << "[CavityWriter] post_lo_hi" << endl;
+    //global_log->info() << "[CavityWriter] post_lo_hi" << std::endl;
```

Comment:
Yes, probably it will never be used and was just for debugging. But I think, there are [tons of those code pieces](https://github.com/ls1mardyn/ls1-mardyn/security/code-scanning?query=is%3Aopen+branch%3Amaster+rule%3Acpp%2Fcommented-out-code).

Reply:
Okay I think we can just ignore it all then :)